build.gradle

```diff
@@ -262,15 +262,16 @@ pitest {
   excludedClasses = [
     'uk.gov.hmcts.reform.cwrdapi.config.*',
     'uk.gov.hmcts.reform.cwrdapi.controllers.*',
-    'uk.gov.hmcts.reform.cwrdapi.CaseWorkerRefApiApplication'
+    'uk.gov.hmcts.reform.cwrdapi.CaseWorkerRefApiApplication',
+    'uk.gov.hmcts.reform.cwrdapi.service.impl.CaseWorkerDeleteServiceImpl'
```

Comment:
The unit tests for this class do not cover all the code, hence it brings down the mutation coverage. Therefore, since this class contains non production code only, @shreedhar-hmcts suggested to exclude it from the mutation tests.

Reply:
That makes sense, thanks for clarifying 👍🏼